site/gatsby-site/src/components/checklists/ExportDropdown.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { Dropdown } from 'flowbite-react';
+import { exportJson } from 'utils/checklists';
+
+const ExportDropdown = ({ checklist }) => {
+  return (
+    <Dropdown label="Export">
+      <Dropdown.Item onClick={() => exportJson(checklist)}>
+        <Trans>JSON</Trans>
+      </Dropdown.Item>
+      <Dropdown.Item onClick={() => alert('Coming soon')}>
```

Comment:
Cool beans. Create an issue to track this feature, HTML + CSV exports, please?

Reply:
https://github.com/responsible-ai-collaborative/aiid/issues/2322